covid-19-support/public/index.html

```diff
@@ -76,6 +76,9 @@
     <script async src="https://www.googletagmanager.com/gtag/js?id=<%= settings.GA_ID %>"></script>
     <% } %>
 
+    <!-- font awesome cdn -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
```

Comment:
@codetheorem - You don't need to include Font Awesome as version 5.0.3 is already included within the solution.